spec/support/boolean_matcher.rb

```diff
@@ -0,0 +1,5 @@
+RSpec::Matchers.define :be_boolean do
+  match do |actual|
+    !!actual == actual
+  end
+end
```

Comment:
We actually want boolean methods, not fake ones.

For instance look at https://github.com/Fullscreen/yt/blob/master/lib/yt/models/snippet.rb#L44-L47